### PR TITLE
Selfie: Fix top bar in video mode

### PR DESCRIPTION
**Pull request by Dhina17**

### CODE REVIEW

**Comment by SebaUbuntu** [comment]
Screenshot_20220810-200944_Camera
Looks fine?

**Comment by luk1337** [comment]
Not sure what this is about, but I just reformatted your commit msg.

**Comment by SebaUbuntu** [comment]
> Screenshot_20220810-200944_Camera Looks fine?

(this is w/o this commit btw)
Are you testing on a 16:9 device?

**Comment by Dhina17** [comment]
> Not sure what this is about, but I just reformatted your commit msg.

For me, in the video mode, top bar is hidden under the PreviewView.

**Comment by SebaUbuntu** [comment]
> > Not sure what this is about, but I just reformatted your commit msg.
> 
> For me, in the video mode, top bar is hidden under the PreviewView.

Mind sending a screenshot? Unless you're testing with my aspect ratio commits we're using a ConstraintLayout so it shouldn't happen at all

**Comment by Dhina17** [comment]
> > Screenshot_20220810-200944_Camera Looks fine?
> 
> (this is w/o this commit btw) Are you testing on a 16:9 device?

with my Redmi 7. Not sure . let me send a video without this commit

**Comment by Dhina17** [comment]
> > > Not sure what this is about, but I just reformatted your commit msg.
> > 
> > 
> > For me, in the video mode, top bar is hidden under the PreviewView.
> 
> Mind sending a screenshot? Unless you're testing with my aspect ratio commits we're using a ConstraintLayout so it shouldn't happen at all

Check this video. https://imgur.com/a/2XrnquW
I haven't tested your aspect ratio commit yet

**Comment by Dhina17** [comment]
May be I should move the view as you did in your staging branch instead of adding elevation

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -8,13 +8,81 @@
...
+        app:layout_constraintBottom_toBottomOf="@+id/viewFinder"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent" />
 
     <androidx.constraintlayout.widget.ConstraintLayout
         android:id="@+id/topButtonsLayout"
         android:layout_width="0dp"
         android:layout_height="wrap_content"
         android:paddingStart="16dp"
-        android:paddingTop="8dp"
```
Comment (luk1337) [inline app/src/main/res/layout/activity_main.xml:17]: items aren't vertically centered after doing that.

**Comment by SebaUbuntu** [comment]
With aspect ratio commits and 16:9 mode
Before:
Screenshot_20220810-214533_Camera
After:
Screenshot_20220810-214346_Camera

It covers a part of the preview view, I don't think that's a good thing

**Comment by Dhina17** [comment]
with the new UI, No need for this anymore.

**Pull request closed without merging**